server/src/main/java/org/opensearch/indices/replication/SegmentReplicationTargetService.java

```diff
@@ -503,7 +521,6 @@ public void onResponse(Void o) {
 
             @Override
             public void onFailure(Exception e) {
-                logger.error(() -> new ParameterizedMessage("Exception replicating {} marking as failed.", target.description()), e);
```

Comment:
This log is also redundant, .fail call below will report the failure to the original listeners.